hw/ip/otbn/dv/model/otbn_model.cc

```diff
@@ -514,15 +514,15 @@ void OtbnModel::reset() {
     iss->reset(has_rtl());
 }
 
-int OtbnModel::send_lc_escalation() {
+int OtbnModel::send_err_escalation(svLogicVecVal *err_val /* logic [31:0] */) {
```

Comment:
This is `bit [31:0]` in the SV DPI code (good!), so the type needs to change to `svBitVecVal` here and line 523 can just use `err_val[0]`.

Can you update the function signatures and comments for this and `otbn_model_send_err_escalation` (and in `otbn_model_dpi.h` too)?